src/components/Info/Info.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import './Info.scss';
+
+type Props = {
+  onHandleChange: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  itemWidth: number,
+  frameSize: number,
+  step: number,
+  animationDuration: number,
+};
+
+export const Info: React.FC<Props> = ({
+  onHandleChange,
+  itemWidth,
+  frameSize,
+  step,
+  animationDuration,
+}) => {
+  return (
+    <div className="info">
+      <ul className="info__list">
+        <li>
+          <label
+            className="info__label"
+            htmlFor="itemWidth"
+          >
+            Item Width :
+            <br />
+            <input
+              type="number"
+              id="itemWidth"
+              min={30}
+              max={260}
+              step={10}
+              value={`${itemWidth}`}
+              onChange={(event) => onHandleChange(event)}
+              name="itemWidth"
+            />
+          </label>
+
+        </li>
+
+        <li>
+          <label
+            className="info__label"
+            htmlFor="frameSize"
+          >
+            Frame Size :
+            <br />
+            <input
+              type="number"
+              id="frameSize"
+              min={1}
+              max={3}
+              value={`${frameSize}`}
+              onChange={(event) => onHandleChange(event)}
```

Comment:
Will works same but less callbacks - fix all similar cases
```suggestion
              onChange={onHandleChange}
```